src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -100,7 +100,8 @@ def _match_offers_bids(self):
         if clearing is None:
             return
         clearing_rate, clearing_energy = clearing
-        log.info(f"Market Clearing Rate: {clearing_rate} & Clearing Energy: {clearing_energy} ")
+        log.info(f"[AREA]: {self.owner.name} ||| [ENGINE]: {self.name} ||| MCR: {clearing_rate} "
```

Comment:
Personal preference, but I would suggest to change these to lowercase, furthermore I was under the impression that some of these are already covered by the generic logger of IAAEngine (like the owner for instance), don't they?